docs/Project-Configuration/Project-Commands.md

```diff
@@ -379,10 +381,18 @@ The parameters for each module are:
 -   `prefix`: the subdirectory to clone the repository in. It will be
     the repository name as a top-level directory in `dir` if omitted
 -   `ref`: must be a commit hash, takes precedence over the `branch`
-    parameter if both specified
+    parameter if both specified (for commits)
 -   `branch`: must be the name of branch, commit hashes _are not
     accepted_.
 
+
+More specifically, module hash priority is as follows:
+* For commit queue patches, we always use the module branch name, to ensure accurate testing.
```

Comment:
nit: Instead of saying a nebulous "we", should we refer to it as "Evergreen" instead? I think of the Evergreen CI system as a single entity at least.

Reply:
Haha fair -- I can rework